docs: update README.md setup order (#72)
PandasWhoCode/version2: README.md

## 💻 Setup
- Authentication through `gh auth login`
- Set the appropriate token permissions: `gh auth refresh --scopes read:project`

>   - Note: The team must be a member of the Project Board in order to view the issues on the board.

Reviewer: ```suggestion
  - Note: The team must have `read` permissions on the Project Board in order to view the issues on the board.
```
